### PR TITLE
feat: homepage added

### PR DESCRIPTION
**Pull request by som749898**
Please review my pr

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  color: var(--white-color);
+}
+
+.shop-button {
+  font-size: 1rem;
+  margin-top: 1.25rem;
+}
+
+.background-dark-grey {
+  background-color: #112233;
```
Comment (HarshithaKonduru214) [inline css/style.css]: Try using variables.

**Review by HarshithaKonduru214 — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            <li><a href="https://www.linkedin.com/in/manishsom20/"><i class="fab fa-linkedin-in"></i></a></li>
+          </ul>
+          <p>&copy; 2022 SKY Designs</p>
+        </footer>
+      </div>
+    </div>
+  </div>
+
+
+
```
Comment (rajkishancode) [inline index.html]: remove extra spaces

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,99 @@
+.container {
+  max-width: 93.75rem;
+  padding: 0 0.625rem;
+}
+
+.home-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```
Comment (rajkishancode) [inline css/style.css:9]: you can  put these property in a separate utility class

**Review by rajkishancode — APPROVED** [review]
left comment . overall look good to me

File: css/style.css (hunk 1 above)
Comment (som749898) [reply css/style.css]: fixed it

File: index.html (hunk 2 above)
Comment (som749898) [reply index.html]: fixed it

**Pull request merged**